source/components/molecules/HelpButton/HelpButton.js

```diff
@@ -86,16 +87,9 @@ const HelpButton = (props) => {
 
   return (
     <>
-      <Modal
-        visible={modalVisible}
-        animationType="slide"
-        onRequestClose={() => {
-          setModalVisible(false);
-        }}
-        presentationStyle="pageSheet"
-      >
+      <Modal visible={visible} hide={toggleModal}>
```

Comment:
`modalVisibilityState={isModalVisible}` maybe?

Reply:
I think it's clearer as is. No need for long names if a concise one suffices.

Reply:
Changed the state name to **isModalVisible**. 
But I want to keep the prop **visible** name so it uses the same prop name as the native component: https://reactnative.dev/docs/modal#visible

Reply:
i vote for visible={isModalVisible} since visible is a prop on the modal component, so no need to include modal in the prop name :D.

Another idea is to use the word show instead of visible, so showModal true/false. But that's details (y)